search-service/src/main/kotlin/com/egm/stellio/search/temporal/web/TemporalApiResponses.kt

```diff
@@ -32,8 +32,10 @@ object TemporalApiResponses {
         lang: String? = null,
     ): ResponseEntity<String> {
         val baseRepresentation = parseRepresentations(requestParams, mediaType)
-
-        val representation = lang?.let { baseRepresentation.copy(languageFilter = it) } ?: baseRepresentation
+        val representation = lang?.let {
+            baseRepresentation.copy(languageFilter = it, timeproperty = query.temporalQuery.timeproperty.propertyName)
+        }
+            ?: baseRepresentation.copy(timeproperty = query.temporalQuery.timeproperty.propertyName)
```

Comment:
Yes, but it is set from the queryParameters, so this will fail for queryEntitiesViaPost.
It may not be the best way to do it. but it has the advantage of not needed to change the parseRepresentation function.

Reply:
hmm, yes. I guess this is why I did the same for languageFilter...

can you just add a one line comment to explain the why?